### PR TITLE
for for dashboard

### PR DESCRIPTION
**Pull request by lilyanB**
- Have images take up all the space
- Let's reduce the size font of the title if possible because I don't it nice to see Massa Wallet written like this.... wdyt?
Expected: it should be in only 1 line I would say

### CODE REVIEW

File: src/components/DashboardStation/DashboardStation.tsx [hunk 1]
```diff
@@ -18,12 +18,13 @@ export function DashboardStation(props: IDashboardStationProps) {
     'theme-dark',
   );
   const [images, setImages] = useState<ReactNode[]>([]);
+  const customClass = 'h-full w-full';
```
Comment (maxime-carabina) [inline src/components/DashboardStation/DashboardStation.tsx:21]:
```suggestion
  const sizeClass = 'h-full w-full';
```

Comment (lilyanB) [reply src/components/DashboardStation/DashboardStation.tsx]: done

**Review by maxime-carabina — APPROVED** [review]

File: src/components/PluginWallet/PluginWallet.tsx [hunk 2]
```diff
@@ -34,8 +34,11 @@ export function ActivePlugin(props: ActivePluginProps) {
     <>
       {iconActive}
       <div className="w-full py-6 text-f-primary bg-secondary flex flex-col items-center">
-        <div className="w-4/5 px-4 py-2 text-center mas-subtitle min-[450px]:mas-title">
-          {title}
+        <div
+          className="px-4 py-2 lg:h-14 mas-title min-[0px]:text-xl sm:text-4xl
+                        min-[790px]:text-4xl lg:text-2xl xl:text-3xl min-[1330px]:text-4xl"
```
Comment (mazmassa) [inline src/components/PluginWallet/PluginWallet.tsx:39]: min-[790px], 
min-[1330px]

Question: Is there a way to **not** rely on that specific values!? I mean... using at least percentage or calc in function of the screen!? 🤔

Comment (lilyanB) [reply src/components/PluginWallet/PluginWallet.tsx]: I have simplified

Comment (mazmassa) [reply src/components/PluginWallet/PluginWallet.tsx]: Awesome! Tks!

**Review by mazmassa — APPROVED** [review]

**Pull request merged**